.github/workflows/codeql-buildscript.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+make -j$(nproc) lm3s6965evb-test
```

Comment:
Suggestion: have the target it builds passed in to this script and put the lm3s6965evb in the .yml file.